projects/demo/src/app/pages/segmented-control/segmented-control-demo.component.ts

```diff
@@ -1,8 +1,8 @@
 import { IdsIconComponent } from '../../../../../widgets/icon/ids-icon.component';
```

Comment:
Also relative here, should be:
```suggestion
import { IdsIconComponent } from '@i-cell/ids-angular/icon';
```